<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>270</width>
     <height>380</height>
    </rect>
   </property>
   <property name="title">
    <string>Celsius</string>
   </property>
   <widget class="QDial" name="celsiusDial">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>170</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="celsiusLcd">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>270</width>
     <height>380</height>
    </rect>
   </property>
   <property name="title">
    <string>Fahrenheit</string>
   </property>
   <widget class="QLCDNumber" name="fahrenheitLcd">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="fahrenheitDial">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>170</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-200</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
